src/pages/dataElements/form/customFields.tsx

```diff
@@ -48,15 +49,27 @@ export function ColorAndIconField() {
 
 export function DomainField() {
     const name = 'domainType'
-    const aggregateInput = useField(name, { type: 'radio', value: 'AGGREGATE' })
-    const trackerInput = useField(name, { type: 'radio', value: 'TRACKER' })
+    const validate = (value: string) => (!value ? 'Required' : undefined)
```

Comment:
This will change once we implemented proper validation with `zod`. @Birkbjo and I agreed that I'll focus on validation while he'll focus on the details sidebar in the list view, so I'll just add proper test cases alongside validation